api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -364,6 +364,15 @@ type FelixConfigurationSpec struct {
 	// [Default: Off].
 	// +optional
 	BPFLogLevel string `json:"bpfLogLevel" validate:"omitempty,bpfLogLevel"`
+	// BPFLogFilters is a comma-separated list of key=values where the value is
+	// a pcap filter expression and the key is an interface name with 'all'
+	// denoting all interfaces, 'weps' all workload endpoints and 'heps' all host
+	// endpoints. 'ctlb' can take a value of 'on' or 'off' to set whether
```

Comment:
It's not clear to me how `ctlb` fits in here - the filters map is a map of interface name to filter string, right? `ctlb` isn't an interface name, so it's a bit out of place to wedge that in here (at least by my naive understanding)

Reply:
ok, will add another option as we would like to expand it later from on/off to a true filter, which is not going to be a pcap filter